scripts/paged_speculative_inference.py

```diff
@@ -141,12 +148,24 @@
 speculator = None
 if args.speculator_path is not None:
     print("loading speculator")
-    speculator = MLPSpeculator(
-        model.config.emb_dim, 4096, model.config.src_vocab_size, n_predict=3
-    )
-    speculator.load_state_dict(
-        torch.load(args.speculator_path, map_location=device)["model_state"]
-    )
+    if args.speculator_source == "fms":
```

Comment:
I think this is a bit confusing since in most of our code we'd use `get_model(path, source)` where 'source' is meta, fms, or hf, and signifies which weight adapter to use.

Reply:
fixed to use `get_model` and added adapter for hf weights